codespell_lib/data/dictionary.txt

```diff
@@ -21220,14 +21272,34 @@ reguarlize->regularize
 reguarlizer->regularizer
 reguarly->regularly
 reguator->regulator
+regulaer->regular
 regulaion->regulation
+regulamentation->regulation
 regulamentations->regulations
 regulaotrs->regulators
 regulaotry->regulatory
 regularily->regularly
-regulariry->regulary
+regulariry->regularly
+regularlisation->regularisation
+regularlise->regularise
+regularlised->regularised
+regularliser->regulariser
+regularlises->regularises
+regularlising->regularising
+regularlization->regularization
+regularlize->regularize
+regularlized->regularized
+regularlizer->regularizer
+regularlizes->regularizes
+regularlizing->regularizing
+regularlly->regularly
+regulary->regular, regularly,
+regulax->regular
+reguler->regular
+regulr->regular
 regultor->regulator
 regultors->regulators
+regultory->regulatory
 regurlarly->regulary
```

Comment:
AssertionError: error regurlarly: correction regulary is an error itself in the same dictionary file dictionary.txt

Reply:
Another one for the problem matcher...